sensors/bms/setup.py

```diff
@@ -0,0 +1,29 @@
+import os
+from glob import glob
+from setuptools import find_packages, setup
+
+package_name = 'bms'
+
+setup(
+    name=package_name,
+    version='0.0.0',
+    packages=find_packages(exclude=['test']),
+    data_files=[
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+        ('share/' + package_name, ['package.xml']),
+        (os.path.join('share', package_name), glob('launch/*.py')),
+    ],
+    install_requires=['setuptools'],
+    zip_safe=True,
+    maintainer='vortex',
+    maintainer_email='karate.martynas1@gmail.com',
+    description='TODO: Package description',
+    license='TODO: License declaration',
+    tests_require=['pytest'],
+    entry_points={
+        'console_scripts': [
+            'bms_publisher = bms.freya_bms_node:main'
```

Comment:
could you give it the same name as the node, that way people can easily identify which node they want to run withouth checking setup.py file every time they want to launch the code, just a good practice for less confusion :)
ie
freya_bms_node = bms.freya_bms_node:main

alternatively change node name to include publisher to make it even clearer, ie freya_bms_publisher_node.py, and then in setup.py write:
freya_bms_publisher_node = bms.freya_bms_publisher_node:main

Reply:
fixed :D renamed the script as well